burger/toppings/tags.py

```diff
@@ -22,7 +22,8 @@ def act(aggregate, classloader, verbose=False):
             if not path.startswith(prefix) or not path.endswith(suffix):
                 continue
             key = path[len(prefix):-len(suffix)]
-            type, name = key.split("/", 2)
+            idx = key.find("/")
+            type, name = key[:idx], key[idx + 1:]
```

Comment:
For my own reference, the reason this didn't work before is because [`str.split`](https://docs.python.org/3/library/stdtypes.html#str.split)'s second parameter is the maximum number of _splits_ to do, not the maximum number of _split strings_ to produce (unlike [Java's String.split](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#split-java.lang.String-int-)).  So 2 indicates that up to 2 splits can be performed, meaning an array can have up to 3 elements.  Had `split("/", 1)` been used, things would have worked, though `find("/")` is probably clearer.

Reply:
Oh I didn't realize how `split()` worked either, and rather than looking it up I just did the most unga-bunga straightforward solution. TIL